### PR TITLE
🐙 octavia-cli: bump with same version as Airbyte

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Airbyte's API is the core dependency of `octavia-cli`.
As Airbyte API is versioned with the Airbyte platform we should version `octavia-cli` with the Airbyte platform to guarantee compatibility and avoid regressions.

## How
* Add octavia-cli version files to `.bumpversion.cfg`
* Update octavia-cli version to the latest Airbyte version
* Run the octavia-cli publish script from `release_version.sh`

### CODE REVIEW

**Comment by alafanechere** [comment]
Questions: 
* Shall we make `octavia-cli` part of the platform build?
* Could you explain what exactly happens when `./gradlew publish` runs?
* How could run `octavia-cli/publish.sh` from gradle task?

File: tools/bin/release_version.sh [hunk 1]
```diff
@@ -42,3 +42,8 @@ VERSION=$NEW_VERSION SUB_BUILD=PLATFORM ./gradlew clean build
 SUB_BUILD=PLATFORM ./gradlew publish
 VERSION=$NEW_VERSION GIT_REVISION=$GIT_REVISION docker-compose -f docker-compose.build.yaml push
 echo "Completed building and publishing..."
+
+# TODO alafanechere: does SUB_BUILD=OCTAVIA_CLI ./gradlew publish would work?
```
Comment (davinchia) [inline tools/bin/release_version.sh:46]: no need to use gradlew publish

File: octavia-cli/publish.sh [hunk 2]
```diff
@@ -2,8 +2,11 @@
 
 set -ux
 VERSION=$1
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+docker login -u airbytebot -p "${DOCKER_PASSWORD}"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+docker run --privileged --rm tonistiigi/binfmt --install all
```
Comment (davinchia) [inline octavia-cli/publish.sh:8]: leave a comment explaining why we need to install this

File: octavia-cli/publish.sh [hunk 3]
```diff
@@ -2,8 +2,11 @@
...
+docker build --push --tag airbyte/octavia-cli:${VERSION} ${SCRIPT_DIR}
 docker buildx create --name octavia_builder > /dev/null 2>&1
 set -e
```
Comment (davinchia) [inline octavia-cli/publish.sh:11]: nit: this should be top of script.

Comment (ChristopheDuong) [reply octavia-cli/publish.sh:11]: see example: https://github.com/airbytehq/airbyte/blob/280245322edef478bf48b4b71f73378ca451d127/airbyte-integrations/bases/base-normalization/entrypoint.sh#L36

**Review by davinchia — APPROVED** [review]
Looks good. 

Good to merge once we resolve the comments.

**Pull request merged**